README.md

```diff
@@ -352,6 +352,75 @@ This is required for some SQL operations that cannot be run within a transaction
 
 -----------------------------------------------------
 
+#### `pgm.renameType( type_name, new_type_name )`
+
+> Rename a data type - [postgres docs](http://www.postgresql.org/docs/current/static/sql-altertype.html)
+
+**Arguments:**
+- `type_name` _[string]_ - name of the type to rename
+- `new_type_name` _[string]_ - name of the new type
+
+-----------------------------------------------------
+
+#### `pgm.addTypeAttribute( type_name, attribute_name, attribute_type )`
+
+> Adds an attribute to data type - [postgres docs](http://www.postgresql.org/docs/current/static/sql-altertype.html)
+
+**Arguments:**
+- `type_name` _[string]_ - name of the type
+- `attribute_name` _[string]_ - name of the attribute to add
+- `attribute_type` _[string]_ - type of the attribute to add
+
+-----------------------------------------------------
+
+#### `pgm.dropTypeAttribute( type_name, attribute_name, options )`
+
+> Adds an attribute to data type - [postgres docs](http://www.postgresql.org/docs/current/static/sql-altertype.html)
```

Comment:
Looks like copy / paste comment. The function name is 'drop...', the comments says 'Adds ...'.